### PR TITLE
fix: Remove duplicate declarations and library

### PR DESCRIPTION
**Pull request by Cal-L**
## **Description**

This small PR removes duplicates from:
- Type declarations
- Library from package.json

## **Related issues**

Fixes:

## **Manual testing steps**

This PR is only reliant on smoke E2E tests

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained what problem this PR is solving and how it is solved.
- [ ] I've linked related issues
- [ ] I've included manual testing steps
- [ ] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by sethkfman — DISMISSED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -238,7 +238,6 @@
     "base-64": "1.0.0",
     "bignumber.js": "^9.0.1",
     "buffer": "5.2.1",
-    "cockatiel": "^3.1.1",
```
Comment (Gudahtt) [inline package.json:241]: This is used by the `assets-controllers` patch, see the note here about it: https://github.com/MetaMask/metamask-mobile/blob/2cf93fc15634de2c93e472a1bbadf8e16069820d/.depcheckrc.yml#L6

I'd encourage using `depcheck` to find unused dependencies, and using that config to track cases where there is a reason for including a dependency that seems unused.

Comment (Gudahtt) [reply package.json:241]: Oh nevermind, I see that it's listed here twice. Didn't realize that was possible without it throwing an error upon install! TIL.

Comment (Gudahtt) [reply package.json:241]: Though, I'm noticing that this entry is in alphabetical order, but the second instance is not. Probably better to remove the second one, as Yarn with alphabetize this list every time a new dependency is added anyway (via `yarn add`), so leaving the dependencies listed out-of-order will cause churn later on.

Comment (Cal-L) [reply package.json]: Updated!

**Review by sethkfman — APPROVED** [review]
LGTM

**Pull request merged**